connectors/src/api/webhooks/webhook_github.ts

```diff
@@ -250,6 +254,14 @@ const _webhookGithubAPIHandler = async (
             jsonBody.pull_request.number,
             res
           );
+        } else if (jsonBody.action === "closed") {
+          return syncCode(
+            enabledConnectors,
+            jsonBody.organization.login,
+            jsonBody.repository.name,
+            jsonBody.repository.id,
+            res
+          );
         } else {
           assertNever(jsonBody.action);
```

Comment:
Unrelated to this PR => we throw for every event that's not opened/edited/closed?

Reply:
We don't go in that code if it's not an event we want to handle :+1: